public/locales/en/translation.json

```diff
@@ -66,7 +66,7 @@
     "saved_query_name_tooltip": "Click to use as Chart Title"
   },
   "widget_customization_format_value_settings": {
-    "section_title": "Value Format",
+    "section_title": "Values Format",
```

Comment:
according to https://metakeen.atlassian.net/browse/FEE-777 this should be `Value Format`